apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/ApolloJsonValueProcessor.java

```diff
@@ -40,6 +42,15 @@ public ApolloJsonValueProcessor() {
     springValueRegistry = SpringInjector.getInstance(SpringValueRegistry.class);
   }
 
+  @Override
+  public Object postProcessBeforeInitialization(Object bean, String beanName)
+      throws BeansException {
+    if (configUtil.isAutoUpdateInjectedSpringPropertiesEnabled()) {
```

Comment:
This is unnecessary and wrong, disable the auto update feature doesn't mean not to inject the `ApolloJsonValue` field or method.